app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.kt

```diff
@@ -99,8 +101,22 @@ fun markupHiddenUrls(context: Context, content: CharSequence): SpannableStringBu
         val start = spannableContent.getSpanStart(span)
         val end = spannableContent.getSpanEnd(span)
         val originalText = spannableContent.subSequence(start, end)
-        val replacementText = context.getString(R.string.url_domain_notifier, originalText, getDomain(span.url))
+        val replacementText = view.context.getString(R.string.url_domain_notifier, originalText, getDomain(span.url))
         spannableContent.replace(start, end, replacementText) // this also updates the span locations
```

Comment:
Is this then still needed?

Reply:
Because the string is no longer translateable? Yes it could be defined right in the code as well, I don't think it matters a lot. Should I change it?

Reply:
One of them didn't seem to be use anymore (lateron in the method). But I probably looked wrong..